### ISSUE TITLE
Your device is taking so long time to reboot. 

### ISSUE THREAD

**Issue by Sanees777** [TeamShinkansen/Hakchi2-CE#333]
I want to switch from the old hackchi 2 (cluster) to the new (ce).  How to do it.  I have this error during installation, although the firewall does not ache, and the virus does not say anything.  What to do?

**Comment by impeeza** [comment]
Your problem is the kernel.  You must to backup all your saves (RTM)  REMOVE all the old Hakchi out your mini as factory. Then install new hakchi and kernel and then restore your saves.

**Comment by Sanees777** [comment]
Please, in more detail, by stages

**Comment by TheTr25** [comment]
#328 see this thred. i got video how to back to stock

**Comment by horrorlass** [comment]
my device is also doing this. I try to flash the kernel and it gets to 11% then tries to reboot my system. it them gives me this message.

**Comment by cutie-suzuki** [comment]
Same with mine. It gets to about 14% and says "Executing Command boota4740000" and then the reboot loading bar comes up, and I'm met with "Your device is taking too long..."

**Comment by DanTheMan827** [comment]
Are you using Hakchi, but you're getting a message saying "Your system is taking too long to reboot"? Try the following suggestions:

- Use the original USB Cable if using a NES/SNES Classic. The OEM cable has a white sticker.
- Turn off Windows Firewall
- Turn off your VPN (if you have one) (In some instances, VPN may need uninstalled)
- Turn off WiFi (if using a laptop)
- Turn off your Antivirus (Uninstall sometimes necessary, if conflicts arise that prevent hakchi usage...especially if "additional services" are installed)
- If using a Virtual Machine on a Mac/Linux, go to USB Settings and enable in both FEL/Powered On Mode
- Also, If using Virtual Machine, make sure you have the virtual box extension pack and usb set to at least 2.0
- You may need to install or update RNDIS Drivers
- Unplug power from back of system, wait 30 seconds, plug back in, try again!
- Restart Computer and/or hakchi

**Comment by meditaknorr** [comment]
I had the same problem...

So far, what I did was:
1. Downloaded H. Ce (Portable)
2. Copied it to a USB Flash to run it on my other PC (a little bit cleaner than my main PC that I use for Software Development...)
3. On my other Clean PC, I entered the H. Ce folder and went into "Drives", and installed the "classic_nintendo.exe";
4. Turned all Firewall OFF, Antivirus OFF, Add The H. CE .exe File to Firewall...
5. Run H. Ce .exe with internet for the first time (it will check for new H. Ce and other things...)
6. Once it is full loaded and open, turn OFF internet, connect your Nintendo Mini or Sega Mini in FEL mode, backup your original Kernel...
7. Install Custom Kernel...
8. Sync selected games with your mini...


Try and say something, it worked here with H. Ce

If still facing issues, we can try use "AnyDesk"  to try run together.
Remote Assistant.


Device: **SNES Mini Classic EU**
OS: **Windows 11 Pro**
Hakchi Version: **Hakchi2 CE v3.9.3** Portable

**Comment by Sanees777** [comment]
Hello everyone, thanks for your replies. I'm sorry I haven't responded here for a while. What helped me was restoring the kernel. I didn't notice at the time, but when I pressed Shift in the "Kernels" parameter, the "Uninstall" button changed to "Uninstall and flash kernel file." Then I installed the original system kernel, and the console started working again. Then I installed the custom kernel, loaded other games, and everything worked fine.
